examples/cpp/dds/DynamicHelloWorldExample/HelloWorldSubscriber.h

```diff
@@ -106,6 +111,14 @@ class HelloWorldSubscriber
 
         uint32_t n_samples;
 
+        std::mutex types_mx;
```

Comment:
Nitpick: usually the members include a final underscore (as is the case with `subscriber_`)